.bash_prompt

```diff
@@ -57,6 +57,9 @@ else
   sexy_bash_prompt_symbol_color="" # NORMAL
 fi
 
+# Define the default prompt terminator character '$'
+sexy_bash_prompt_symbol='$'
```

Comment:
Note that with double-quotes, if a user edits the `prompt_symbol` to add anything after the `"$"`, bash will incorrectly interpret it as a substitution and the output will likely be zero-length. Would you prefer `"\$"`?

You make a good point about preserving backward compatibility. I'll add a commit to calculate the default depending on UID, but only at runtime.